### DISCUSSION TITLE
Can't remove faceplate on create3. Need help.

### DISCUSSION THREAD

**Discussion by ctgitxb** [iRobotEducation/create3_docs#151] [Questions] [answered]
Does anyone have a method to remove the create3 faceplate when it is stuck?   I had never removed the faceplate but now would like to open the create3 to connect a raspberry pi 4. The method shown does not work. I put on gloves as well to really push counter clockwise but it will not move. Really locked in since manufacture. Is there a safe way to use a tool to apply more leverage?

**Comment by shamlian** [comment, marked as answer]
You can use a tool to tap on the tab help unstick things; we have a video of this posted; stand by while I look for the link. In the mean time, the adapter board comes out of the robot, if you are just trying to get access to the switch; just gently pull it out through the rear. It can help to stick a fingernail under the lip on the bottom edge.

**Reply by ctgitxb** [reply]
Thank you! Used a rubber mallet and a magic marker as in video.

**Reply by shamlian** [reply]
Link to video for future searchers: https://youtu.be/UeLHrAvZ_h0?t=210
